src/utils.js

```diff
@@ -0,0 +1,407 @@
+import _ from 'lodash';
```

Comment:
There's a lot of LCO specific logic in here. Hopefully that goes away once we switch to the component lib compose page and remove some of the more LCO specific plots/pages...